heareval/tasks/dcase2016_task2.py

```diff
@@ -117,10 +117,7 @@ def get_requires_metadata(self, split: str) -> pd.DataFrame:
             )
             metadata = metadata.assign(
                 relpath=sound_file,
-                slug=lambda df: df.relpath.apply(self.slugify_file_name),
-                subsample_key=self.get_subsample_key,
                 split=lambda df: split,
-                split_key=self.get_split_key,
```

Comment:
I accidentally remove split_key in this PR. I fix it in https://github.com/neuralaudio/hear-preprocess/pull/8